hnn_core/params.py

```diff
@@ -24,28 +26,117 @@ def _count_evoked_inputs(d):
     return nprox, ndist
 
 
-class Params(dict):
-    """Params object.
+def _read_json(fname):
+    """Read param values from a .json file.
+    Parameters
+    ----------
+    fname : str
+        Full path to the file (.json)
+
+    Returns
+    -------
+    params_input : dict
+        Dictionary of parameters
+    """
+    with open(fname) as json_data:
+        params_input = json.load(json_data)
+
+    return params_input
 
+def _read_legacy_params(fname):
+    """Read param values from a .param file (legacy).
+    Parameters
+    ----------
+    fname : str
+        Full path to the file (.param)
+
+    Returns
+    -------
+    params_input : dict
+        Dictionary of parameters
+    """
+
+    params_input = {}
+    with open(fname, 'r') as fp:
+        ln = fp.readlines()
+        for l in ln:
+            s = l.lstrip()
```

Comment:
normally we try to avoid single letter variable names because they are hard to Ctrl + F. But since you copied this verbatim from old code, I can live with it

Reply:
fixed